### PR TITLE
Updating to MUI 5

### PR DESCRIPTION
**Pull request by jaruesink**
- ThemeBase needs to be utilized with MUI in all projects currently (provides styles to the MUI components)
- New masking library (from the MUI documentation)
- I added some documentation to the theming page, let me know if it makes sense.

### CODE REVIEW

**Comment by jaruesink** [comment]
@jaredhill4 did you have anything in flux with the component library?

**Comment by jaruesink** [comment]
I can't reproduce the tooltip screenshot difference, styles probably were delayed loading or something. 
https://percy.io/b2b04a4a/lc-component-library/builds/14380868/changed/812405668?browser=chrome&browser_ids=19%2C20%2C21&subcategories=unreviewed%2Cchanges_requested&viewLayout=side-by-side&viewMode=new&width=1280&widths=1280

File: packages/ui-components/src/lib/inputs/InputSelect/InputSelect.tsx [hunk 1]
```diff
@@ -170,7 +171,7 @@ export const InputSelect: FC<InputSelectProps> = ({
     PaperComponent: props => <Paper className="lc-input-select-paper" {...props} />,
     getOptionLabel: (option: { [key: string]: any }) => _get(option, optionLabelKey) || '',
     getOptionDisabled: option => option.isDisabled,
-    getOptionSelected,
+    // getOptionSelected,
```
Comment (dwene) [inline packages/ui-components/src/lib/inputs/InputSelect/InputSelect.tsx:174]: should we remove this?

File: workspace.json [hunk 2]
```diff
@@ -75,7 +75,7 @@
     "ui-components-e2e": {
       "root": "packages/ui-components-e2e",
       "sourceRoot": "packages/ui-components-e2e/src",
-      "projectType": "application",
+      "projectType": "aplication",
```
Comment (jaruesink) [inline workspace.json:78]:
```suggestion
"projectType": "application",
```

File: packages/ui-components/src/lib/inputs/InputSelect/InputSelect.tsx (hunk 1 above)
Comment (jaruesink) [reply packages/ui-components/src/lib/inputs/InputSelect/InputSelect.tsx:174]: uh nope, thanks!

**Pull request merged**